lib/src/kvs/tests/raw.rs

```diff
@@ -208,7 +208,9 @@ async fn scan() {
 	tx.commit().await.unwrap();
 	// Create a readonly transaction
 	let mut tx = ds.transaction(Read, Optimistic).await.unwrap();
-	let val = tx.scan("test1".."test9", u32::MAX).await.unwrap();
+	let val =
+		tx.scan_paged(ScanPage::from("test1".into().."test9".into()), u32::MAX).await.unwrap();
```

Comment:
Checking the code coverage, it seems that `tx.scan()` is not tested anymore.
Shall we duplicate the test so both tx.scan and tx.scan_paged are tested?

<img width="545" alt="Screenshot 2024-01-09 at 13 41 31" src="https://github.com/surrealdb/surrealdb/assets/1282276/66729428-8cc5-4c29-8699-86f2a21ff799">

Reply:
Reverted raw test for scan and used same test for scan_paged underneath. Thanks for spotting!